### DISCUSSION TITLE
Scap Security Guide for Amazon Linux 2/ 2023 versions

### DISCUSSION THREAD

**Discussion by prasannakumarkn** [ComplianceAsCode/content#10999] [Q&A] [answered]
I have fetched the latest release for SSG - scap-security-guide-0.1.69 and however I am not seeing AmazonLinux2 SSG, can we use any alternative for amazon linux? if not, please  let us know if there is any support for AmazonLinux from future releases?

**Comment by marcusburghardt** [comment, marked as answer]
Currently there is no support for Amazon Linux in the project and no plans to include it. But contributions to support and maintain it in the project would be welcome.

**Reply by hipponix** [reply]
@marcusburghardt happy to help here.
May I ask you to confirm that nothing has been done and that none of the existing products can be utilized for Amazon Linux ?
It seems like al2023 is based and derived from Fedora than I wonder if we could easily adopt Fedora product .. or perhaps RH or any other variations?
Otherwise, happy to help and contribute.
Best
Mirco

**Reply by marcusburghardt** [reply]
Great @hipponix . The situation is still the same.
You can get some references of similar products already in the project here:
https://github.com/ComplianceAsCode/content/tree/master/products

And here is a documentation about adding new products to the project:
https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#creating-a-new-product

Finally I also would recommend that you join our Gitter channel, where most of the maintainers are present:
https://matrix.to/#/#Compliance-As-Code-The_content:gitter.im
